Buildings/Resources/src/ThermalZones/Detailed/Validation/BESTEST/simulateAndPlot.py

```diff
@@ -962,25 +974,40 @@ def update_html_tables(comDat):
         f.write(newMoContent)
 
 def _generate_load_tables(comDat, allTools, lessTools):
+    withLimits = '''
+<tr>
+<th>Case</th>
+<th>LowerLimits</th>
+<th>UpperLimits</th>
```

Comment:
Here better use real grammar, e.g., "Lower limit" and "Upper limit".

Reply:
Updated.